### PR TITLE
Refactor mysql fields naming

### PR DESCRIPTION
**Pull request by ruflin**
* Complete fields.yml

### CODE REVIEW

File: metricbeat/tests/system/test_mysql.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import metricbeat
+from nose.plugins.attrib import attr
+
+MYSQL_FIELDS = metricbeat.COMMON_FIELDS + ["mysql"]
+
+MYSQL_STATUS_FIELDS = ["clients", "cluster", "cpu", "keyspace", "memory",
+                     "persistence", "replication", "server", "stats"]
+
+
+class RedisInfoTest(metricbeat.BaseTest):
```
Comment (tsg) [inline metricbeat/tests/system/test_mysql.py]: Usually I prefer calling these just Test so they are easy to reference in the CLI and we don't forget to update them when copying code around :-).

File: metricbeat/tests/system/test_mysql.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @attr('integration')
+    def test_status(self):
+        """
+        Redis module outputs an event.
```
Comment (tsg) [inline metricbeat/tests/system/test_mysql.py]: s/Redis/MySQL

File: metricbeat/tests/system/test_mysql.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+        output = self.read_output_json()
+        self.assertEqual(len(output), 1)
+        evt = output[0]
+
+        self.assertItemsEqual(self.de_dot(MYSQL_FIELDS), evt.keys())
+        mysql_info = evt["mysql"]["status"]
+        #self.assertItemsEqual(self.de_dot(REDIS_INFO_FIELDS), redis_info.keys())
+        #self.assertItemsEqual(self.de_dot(CLIENTS_FIELDS), redis_info["clients"].keys())
+        #self.assertItemsEqual(self.de_dot(CPU_FIELDS), redis_info["cpu"].keys())
```
Comment (tsg) [inline metricbeat/tests/system/test_mysql.py]: These could be cleaned up, right?

File: metricbeat/tests/system/test_mysql.py (hunk 1 above)
Comment (ruflin) [inline metricbeat/tests/system/test_mysql.py]: renamed ...

File: metricbeat/tests/system/test_mysql.py (hunk 2 above)
Comment (ruflin) [inline metricbeat/tests/system/test_mysql.py]: so much about copy paste

File: metricbeat/tests/system/test_mysql.py (hunk 3 above)
Comment (ruflin) [inline metricbeat/tests/system/test_mysql.py]: done

**Pull request merged**